plugins/prefixIds.js

```diff
@@ -146,6 +146,10 @@ exports.fn = function(node, opts, extra) {
         if(prefix === false){
             return escapeIdentifierName(name);
         }
+        if (name.startsWith(escapeIdentifierName(prefix) + opts.delim)) {
```

Comment:
You should combine this check with the  previous one since they return the same value.

Reply:
@GreLI @XhmikosR Thanks! Solved